securitycenter/assets/main_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Smoke tests for main.go
+
+package main
+
+import (
+	"os"
+	"testing"
+	"time"
+)
+
+func setup(t *testing.T) string {
+	orgId := os.Getenv("GCLOUD_ORGANIZATION")
+	if orgId == "" {
+		t.Skip("GCLOUD_ORGANIZATION not set")
+	}
+	return orgId
+}
+
+func TestAllAssetsReturned(t *testing.T) {
```

Comment:
The test function is usually named after the function being tested, rather than the behavior being verified. So, the test for `func foo()` would be `func TestFoo(...)`. In this case, it would be `TestListAllAssets`. This makes it easier to debug and develop later because its clear which function this test is testing.